src/Php73/bootstrap.php

```diff
@@ -13,4 +13,12 @@
     if (!function_exists('is_countable')) {
         function is_countable($var) { return is_array($var) || $var instanceof Countable || $var instanceof ResourceBundle || $var instanceof SimpleXmlElement; }
     }
+
+    if (!function_exists('array_key_first')) {
+        function array_key_first(array $array) { foreach ($array as $key => $value) { return $key; } }
+    }
+
+    if (!function_exists('array_key_last')) {
+        function array_key_last(array $array) { return key(array_slice($array, -1, 1, true)); }
```

Comment:
I'd suggest to go with a foreach here also:
`$key = null; foreach ($array as $key => $value); return $key;`